.golangci.yml

```diff
@@ -19,7 +19,7 @@ linters-settings:
     min-complexity: 20 # default is 30 which is too high
   goheader:
     template: |-
-      Copyright © {{ year }} SUSE LLC
+      Copyright © SUSE LLC
```

Comment:
not sure about thins, doesnt copyright needs to include the year for some weird reason?

Reply:
Hehe, we are not alone: https://daniel.haxx.se/blog/2023/01/08/copyright-without-years/

Reply:
🤔  googling around, found some statements that the year (at least the first year of publication) may be needed to compute the duration of the copyright. To make things more complicated, seems different countries may have different rules. Will try to get some advice from the legal dept.